embed/extmod/modtrezorio/modtrezorio-flash.h

```diff
@@ -5,13 +5,7 @@
  * see LICENSE file for details
  */
 
-#if defined TREZOR_STM32
-#include "flash.h"
-#elif defined TREZOR_UNIX
-#include "unix-flash-mock.h"
-#else
-#error Unsupported TREZOR port. Only STM32 and UNIX ports are supported.
-#endif
+#include "../../trezorhal/flash.h"
```

Comment:
Do we need relative path here?

Reply:
trezorhal is not in include path for the unix port, but we can add it.

Reply:
Ah, right. Let's keep it that way.